pkg/operator/ceph/object/bucket/util.go

```diff
@@ -47,12 +48,13 @@ const (
 	objectStoreName      = "objectStoreName"
 	objectStoreNamespace = "objectStoreNamespace"
 	objectStoreEndpoint  = "endpoint"
+	threadcount          = "5"
```

Comment:
`bucketProvisionerNumThreads` is a more specific name and allows other operators to use a different thread count